nextjs-blog/pages/post/[id].tsx

```diff
@@ -12,7 +12,7 @@ export const getServerSideProps = async (
       id: true,
       title: true,
       content: true,
-      filter: e.op(post.id, '=', e.uuid(context!.params!.id as string)),
+      filter_single: e.op(post.id, "=", e.uuid(context!.params!.id as string)),
```

Comment:
We could also update this to the new object style syntax (but the `e.op` way is also still valid).
```ts
filter_single: {
  id: context!.params!.id as string
}
```

Reply:
Thanks for calling that out. I like the new syntax a lot better. I may leave it like this for this iteration since a change might need to ripple out to the video and the guide. Next time we revisit or if we produce a new guide/example for another framework, I'll try to remember to use this instead.